test_runner/src/main/kotlin/ftl/config/android/AndroidGcloudConfig.kt

```diff
@@ -210,6 +218,7 @@ data class AndroidGcloudConfig @JsonIgnore constructor(
             autoGoogleLogin = FlankDefaults.DISABLE_AUTO_LOGIN
             useOrchestrator = true
             environmentVariables = emptyMap()
+            grantPermissions = null
```

Comment:
I think we should not set `null` here. It will result in printed output `grant-permissions: null` which might be confusing for users. Option description has info about `all` default value.

Reply:
so you assume it should be set to ALL at this point?

Reply:
defaulted

Reply:
I think so, I would be surprised seeing `null` (but it's only my subjective opinion)